tests/stream-creation/asserted-rides-stream.yaml

```
@@ -4,6 +4,7 @@ metadata:
   name: rides
 spec:
   allowDirect: false
+  allowMsgTTL: false
```

```suggestion
  allowMsgTtl: false
```

I think that the test is failing because of different  case used.